runtime/lib/backend/openqasm/OpenQasmRunner.hpp

```diff
@@ -121,6 +121,14 @@ struct BraketRunner : public OpenQasmRunner {
             from braket.aws import AwsDevice
             from braket.devices import LocalSimulator
             from braket.ir.openqasm import Program as OpenQasmProgram
```

Comment:
Looking at this, do we even have a dependency on the amazon-braket-pennylane-plugin ?

Reply:
We never import `amazon-braket-pennylane-plugin`. But we obtain a `class BraketQubitDevice(QubitDevice):` from pennylane itself. Which is defined in `amazon-braket-pennylane-plugin`.

Reply:
I could also make this change in the `runtime.py` file. Somewhere around these lines. 

```python
    if dname == "braket.local.qubit":  # pragma: no cover
        device_kwargs["device_type"] = dname
        device_kwargs["backend"] = (
            # pylint: disable=protected-access
            device._device._delegate.DEVICE_ID
        )
    elif dname == "braket.aws.qubit":  # pragma: no cover
        device_kwargs["device_type"] = dname
        device_kwargs["device_arn"] = device._device._arn  # pylint: disable=protected-access
        if device._s3_folder:  # pylint: disable=protected-access
            device_kwargs["s3_destination_folder"] = str(
                device._s3_folder  # pylint: disable=protected-access
            )
```